### PR TITLE
[WIP] Relax strict versions in requirements

### LINKED ISSUE #360: Relax version number requirements for dependencies?
Hello,

Thanks for your work on this great library!

Would it be possible to relax the version number requirements when installing darts? Currently it seems that all dependencies are pinned to exact versions, but this means it can be a little tricky to integrate into existing workflows / environments.

### PR DESCRIPTION
**Pull request by tomasvanpottelbergh**
Fixes #360 and paves the way for a conda package.

### Summary

Changed exact version numbers in requirements to ranges.

### CODE REVIEW

File: requirements/core.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-scikit-learn==0.24.1
-joblib==1.0.1
-requests==2.25.1
\ No newline at end of file
+numpy>=1.19.0
+xarray>=0.18.0
+scipy>=1.6.0
+statsmodels>=0.12.0
+matplotlib>=3.4.0
+pandas>=1.2.0,<1.3.0
```
Comment (hrzn) [inline requirements/core.txt:6]: any reason why this is upper bounded?

Comment (tomasvanpottelbergh) [reply requirements/core.txt:6]: pandas 1.3.0 broke the tests, something to be investigated by someone more familiar with the codebase :)

**Review by hrzn — APPROVED** [review]
LGTM

**Comment by tomasvanpottelbergh** [comment]
I think it might be good to wait until @ymaktepi has changed the CI to update to the latest packages?

**Comment by hrzn** [comment]
@tomasvanpottelbergh I agree let's wait (it'll probably wait for the next release)

**Comment by tomasvanpottelbergh** [comment]
Included in #380

**Pull request closed without merging**